### ISSUE TITLE
Exclude EE Information by default

### ISSUE THREAD

**Issue by LucaNerlich** [boazpoolman/strapi-plugin-config-sync#88]
Hi,

I believe `/config/sync/core-store.ee_information.json` should be excluded by default. Otherwise the license could be commited in plaintext to the repo by accident.

The json file

```json
{
  "key": "ee_information",
  "value": {
    "lastCheckAt": 1677655928783,
    "license": "my-license-in-plaintext"
  },
  "type": null,
  "environment": null,
  "tag": null
}
```

**Comment by boazpoolman** [comment]
Agreed! Do you want to put up a PR? Otherwise I’ll make the change myself.

Thanks for the donation ❤️
Much appreciated.

**Comment by boazpoolman** [comment]
This was released with v1.1.2
